src/components/Button/Button.tsx

```diff
@@ -10,8 +10,17 @@ import { ButtonBehavior } from '../../lib/accessibility'
 
 /**
  * A button.
- * @accessibility This is example usage of the accessibility tag.
- * This should be replaced with the actual description after the PR is merged
+ * @accessibility
+ * > Button
+ * Default behaviour name: ButtonBehavior
+ * Default behaviour description:
+ * - If element is not button, then "role='button'" is applied.
+ *
```

Comment:
I would start a new section here, something like Accessibility consideretions. but not sure about the wording yet. The goal is to indicate that we are no longer describing behavior, but other accessibility topics related to the component

Reply:
maybe it should be:

Default behavior: ButtonBehavior
- adds role='button' if element type is other than 'button'

Other considerations:
- for disabled buttons, add 'disabled' attribute so that the state is properly recognized by the screen reader
- if button includes icon only, textual representation needs to be provided by using 'title', 'aria-label', or 'aria-labelledby' attributes